video/decode/vd_lavc.c

```diff
@@ -715,6 +715,33 @@ static void init_avctx(struct mp_filter *vd)
     if (!lavc_codec)
         return;
 
+    /*
+     * DEAR DISTRIBUTION PACKAGER,
+     * Before removing the check below to silence the users filing bugs against
+     * your package prompted by it that you can't do anything about, please
+     * consider the hypocrisy of your actions. libopenh264 users frequently
+     * report problems on mpv's issue tracker and IRC channel that we can't do
+     * anything about either.
+     *
+     * Should you choose to go ahead with patching out this check anyway, thus
+     * making upstream mpv's life more difficult, then upstream mpv may choose
+     * to respond in kind by making your life difficult as well.
```

Comment:
There's no need to be this passive aggressive. We'll just tell Fedora/Tumbleweed users to go report bugs to their distro if they package mpv with this warning removed.

Reply:
ok fine sheesh, no fun allowed